taskcluster/ffios_taskgraph/__init__.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+import re
+
+from importlib import import_module
+from six import text_type
+from voluptuous import All, Any, Range, Required
+
+from taskgraph.parameters import extend_parameters_schema
+
+
+def register(graph_config):
+    """
+    Import all modules that are siblings of this one, triggering decorators in
+    the process.
+    """
+    _import_modules(["job"])
```

Comment:
This line has changed compared to https://github.com/mozilla-mobile/fenix/blob/34144fd254f00bd8a4176a296fad384ced69f2bf/taskcluster/fenix_taskgraph/__init__.py#L25

Taskcluster doesn't produce/sign any artifacts, so there is no need for `routes`. We don't have any scriptworkers either, so no need to define `worker_types`. `target_tasks` will be needed soon, but I don't know what it going to look like yet, so I'll add it later.